lib/packwerk/offense_collection.rb

```diff
@@ -71,17 +73,28 @@ def outstanding_offenses
 
     private
 
-    sig { params(package: Packwerk::Package).returns(Packwerk::DeprecatedReferences) }
-    def deprecated_references_for(package)
-      @deprecated_references[package] ||= Packwerk::DeprecatedReferences.new(
+    sig { params(package: Packwerk::Package).returns(Packwerk::PackageTodo) }
+    def package_todo_for(package)
+      @package_todo[package] ||= Packwerk::PackageTodo.new(
         package,
-        deprecated_references_file_for(package),
+        package_todo_file_for(package),
       )
     end
 
     sig { params(package: Packwerk::Package).returns(String) }
-    def deprecated_references_file_for(package)
-      File.join(@root_path, package.name, "deprecated_references.yml")
+    def package_todo_file_for(package)
+      if Pathname.new(@root_path).join(package.name, "deprecated_references.yml").exist?
+        warning = <<~WARNING
```

Comment:
```suggestion
        warning = <<~WARNING.squish
```
Should this be one line?